Frontend/src/app/shared/components/home/home.component.html

```diff
@@ -1,21 +1,17 @@
-<div class="container">
-  <div class="intro-section">
-    <a class="intro-text">Time tracking <br class="text-break">made <span class="colored-word">EASY</span><br>
-    </a>
-    <a class="intro-text-bottom">
-      <span class="small-text">Time tracking software designed for universities. <br>
-      <span class="ext-text">NTIME is a time tracker that helps faculty and students <br> with project time management.
-      </span></span>
-    </a>
-  </div>
-
-  <div class="tracker-example-section">
-    <img src="assets/tracker-example.png" alt="intro" />
-  </div>
+<div class="color-container">
```

Comment:
Instead of adding an additional container for the color here. What you can do is just apply more styles to the `container`. It will use both the values from the imported scss and the components scss.

Reply:
Color container removed